### PR TITLE
Adds a bit more persuasion

### PR DESCRIPTION
**Pull request by sapioit**
At times, making things personal and touching the developers' *pride* has it's merits. This might be one such moment.

### CODE REVIEW

File: WHY.md [hunk 1]
```diff
@@ -22,6 +22,13 @@ So in my point of view the `MAJOR` should only be used to go from `1.1.0` to `2.
 For me something to handle changes that are **_Backwards Incompatible_** is missing between `MAJOR` and `MINOR`.
 
 
+## A different angle
+
+We, as developers, are preaching to write easy to understand code that is more difficutl to misunderstand and uses variable numbers that help enforce that, trying to make the code as explicit as possible. 
+
```
Comment (Exadra37) [inline WHY.md]: you mean `variables names` instead of `variable numbers`?

File: WHY.md [hunk 2]
```diff
@@ -22,6 +22,13 @@ So in my point of view the `MAJOR` should only be used to go from `1.1.0` to `2.
...
+But when it comes to versioning, we're doing the complete opposite of what we're asking others to do. And that creates confusion, which leads to lots of misunderstandings which lead to misuse on a large scale. *Simply put, ilogic.*
```
Comment (Exadra37) [inline WHY.md]: I am not getting the meaning of `Simply put, ilogic.`?

File: WHY.md (hunk 1 above)
Comment (sapioit) [reply WHY.md]: Yeah, sorry. Didn't catch that typo.

File: WHY.md [hunk 3]
```diff
@@ -24,9 +24,9 @@ For me something to handle changes that are **_Backwards Incompatible_** is miss
 
 ## A different angle
 
-We, as developers, are preaching to write easy to understand code that is more difficutl to misunderstand and uses variable numbers that help enforce that, trying to make the code as explicit as possible. 
+We, as developers, are preaching to write easy to understand code that is more difficutl to misunderstand and uses variable names that help enforce that, trying to make the code as explicit as possible. 
 
-But when it comes to versioning, we're doing the complete opposite of what we're asking others to do. And that creates confusion, which leads to lots of misunderstandings which lead to misuse on a large scale. *Simply put, ilogic.*
+But when it comes to versioning, we're doing the complete opposite of what we're asking others to do. And that creates confusion, which leads to lots of misunderstandings which lead to misuse on a large scale. *Simply put, it's ilogic.*
```
Comment (Exadra37) [inline WHY.md]: I think you want to say `it's illogical`... though I needed to google to understand this new word ;)

Comment (Exadra37) [reply WHY.md]: Also can you please move this entire section to end, before **Conclusion**.

When taking a look to the whole document it seems to make more sense their.

Comment (sapioit) [reply WHY.md]: Thanks for pointing it out! Will fix it in a moment.

**Comment by Exadra37** [comment]
@sapioit 

Can you please move this entire section to end, before Conclusion.

When taking a look to the whole document it seems to make more sense their.

File: WHY.md [hunk 4]
```diff
@@ -65,6 +58,13 @@ Why I have changed the names... because if I want a **Semantic Versioning** sche
 So when introducing a change that is **_Backwards Incompatible_** I will go from `1.0.1.1` to `1.1.0.0`, instead of the current [SemVer]((http://semver.org/)) practice of going from `1.1.1` to `2.0.0`.
 
 
+## A different angle
```
Comment (sapioit) [inline WHY.md:61]: Changed the order

**Review by Exadra37 — APPROVED** [review]

**Pull request merged**